mitmproxy/proxy2/layers/http/_hooks.py

```diff
@@ -0,0 +1,66 @@
+from mitmproxy import http
+from mitmproxy.proxy2 import commands
+
+
```

Comment:
This file should probably be rendered in our docs: https://docs.mitmproxy.org/master/addons-events/

Reply:
Yes. We are collecting all hooks [in commands.py](https://github.com/mitmproxy/mitmproxy/pull/4343/files#diff-95a362d2a1ab18ce23d76ccbdb6638bfce2255455f228134a51d86488185fb29R125), there's already a comment that we should autogenerate docs from that.